ocdsdata-cli

```diff
@@ -62,8 +62,8 @@ def main():
             print(" - %s" % source_id)
         quit(-1)
 
-
-    base_dir = args.basedir or os.getcwd()
+    this_dir = os.path.dirname(os.path.realpath(__file__))
+    base_dir = args.basedir or os.path.join(this_dir, "data")
```

Comment:
however this will break when not specifying the basedir and running from a system install or non user-writable directory... (?)